pkgs/development/compilers/llvm/16/libunwind/default.nix

```diff
@@ -40,6 +40,11 @@ stdenv.mkDerivation rec {
     cd ../runtimes
   '';
 
+  postInstall = lib.optionalString (enableShared) ''
+    # libcxxabi wants to link to libunwind_shared.so (?).
+    ln -s $out/lib/libunwind.so $out/lib/libunwind_shared.so
+  '';
+
```

Comment:
I didn’t think this would really affect Darwin since it uses Apple’s implementation of libunwind, but then I noticed recently that Apple dropped their implementation in favor of shipping the one from LLVM (since macOS 11). Making this conditional would allow nixpkgs to also use LLVM’s libunwind on Darwin in the future.

Reply:
Condition now reads:
```nix
postInstall = lib.optionalString (enableShared && !stdenv.hostPlatform.isDarwin) ''
```